upload/catalog/model/catalog/product.php

```diff
@@ -594,4 +594,4 @@ public function getTotalProductSpecials() {
 		}
 	}
 }
-?>
\ No newline at end of file
+?>
```

Comment:
No need for closing tag at all. But hat would be much more global change in all files.